keys/nutritionix.js

```diff
@@ -0,0 +1,17 @@
+const axios = require('axios');
+require('dotenv').config();
+
+
+const nutriAXIOS = axios.create({
+  baseURL: 'https://trackapi.nutritionix.com/v2/natural/nutrients',
+  headers: {
+    'x-app-id': process.env.ID,
+    'x-app-key': process.env.API_KEY,
```

Comment:
put id & key in process.env to make heroku deployment easier on myself. still works.